packages/testing/src/database.ts

```diff
@@ -26,18 +26,27 @@ export async function mockBuilder({
   referralCode = randomString(),
   farcasterId,
   farcasterName,
-  wallets = []
+  wallets = [],
+  weeklyStats = []
 }: Partial<
   Scout & {
     githubUserId?: number;
     githubUserLogin?: string;
     createNft?: boolean;
     nftSeason?: string;
     wallets?: { address: string }[];
+  } & {
```

Comment:
why is there an "} & {" breaking these two inline interfaces up?

Reply:
also just noting i came. up with a slightly different interface to pass in stats when using mockScout. not sure either is better or that they necessarily have to be the same